eclipse-theia/theia
# Notebook Outline-View and Breadcrumbs.

#### What it does

this adds support for the outline view and breadcrumbs for the notebook editor.
Also makes the jupyter `Show Table of contents` menu item working in the main notebook toolbar since its only opening the outline view.

#### How to test

Create or open a notebook and see the breadcrumbs showing all cells.
Trust the workspace and click  `Show Table of contents (Outline View)` to reveal the outline view.
Changing selection in editor or outline view should reflect respectivley 

#### Follow-ups

There is currently no markdown based hirachy yet. That needs to be implemented later. Same as only showing code cells in the breadcrumbs and not in the outline view



#### Review checklist


#### Reminder for reviewers

- As a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
